client/src/pages/AddFood.js

```diff
@@ -58,7 +58,10 @@ function AddFood() {
   return (
     <div className="form">
       <Layout />
-      <Form onSubmit={handleSubmit((data) => handleForm(data))}>
+      <Form
+        onSubmit={handleSubmit((data) => handleForm(data))}
+        style={{ marginTop: "90px", marginLeft: "400px" }}
```

Comment:
Can this styling be moved into the `Form` styled component on line `82` instead?  It would be ideal to keep all the styling in one spot.

Also, are you sure a left margin of 400px is what you want here?  Are you trying to center the form on the screen?

Reply:
Moved it to `Form` styled component..using `position absolute` seemed to work